src/app/options.cpp

```diff
@@ -45,6 +45,10 @@
 
 #include "base/utils/misc.h"
 
+static const int USAGE_INDENTATION = 4;
```

Comment:
There is an unnamed namespace right below. These constants would fit there perfectly.